.github/ISSUE_TEMPLATE/bug_ivv_documentation.md

```diff
@@ -0,0 +1,53 @@
+---
+name: 🐞 Bug IVV
+about: File a bug/issue
```

Comment:
I think we could indicate here as well that the bug is about the documentation.

Reply:
Yes, documentation is added